docs/disciplines/Imaging/nifti.md

```diff
@@ -0,0 +1,34 @@
+# Nifti Format
+
+<https://nipy.org/nibabel/coordinate_systems.html>
+<https://nipy.org/nibabel/nifti_images.html>
+NIFTI first poster: <https://nifti.nimh.nih.gov/nifti-1/documentation/hbm_nifti_2004.pdf>
+NIfTI structure diagram: <https://nifti.nimh.nih.gov/nifti-1/documentation/nifti1diagrams_v2.pdf>
```

Comment:
descriptor and link are not in the same line; if you could add a description to what this is supposed to show, would be useful